src/services/messengerSync/consumer.js

```diff
@@ -8,29 +9,58 @@ const QUEUE_EXTERNAL = Configuration.get('RABBITMQ_MATRIX_QUEUE_EXTERNAL');
 
 let channel;
 
-const validateMessage = (content) => (content.userId && (content.courses || content.teams || content.fullSync));
+const validateMessage = (content) => {
+	if (!content.action) {
+		return false;
```

Comment:
this should test action to be a member of a predefined array

Reply:
create an predefined object like ACTIONS = { SYNC_SCHOOL: 'syncSchool', ... } which can be reused